org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/Application.java

```diff
@@ -84,13 +84,15 @@ public class Application {
   private static final Logger LOG = LoggerFactory.getLogger(Application.class);
 
   protected static final LazyValue<Application> INSTANCE = new LazyValue<>(Application.class);
+  protected static final long START_TIME = System.nanoTime();
 
   protected final AtomicReference<Server> m_server = new AtomicReference<>();
 
   public static void main(String[] args) {
     LOG.info("Starting platform");
     IPlatform platform = Platform.get();
     platform.awaitPlatformStarted();
+    LOG.info("Platform start took {} ms", StringUtility.formatNanos(System.nanoTime() - START_TIME));
 
     LOG.info("Starting application");
     INSTANCE.get().start();
```

Comment:
I would log the duration of the application start here (instead of in the `logServerReady` method). This reduce changes in start methods and do not require to pass variables between methods.

```suggestion
    INSTANCE.get().start();
    LOG.info("Application start took {} ms, total startup (platform/application) took {} ms" ....
```